File: Java/src/classes/StrengthExercise.java
package classes;

public class StrengthExercise extends Exercise {

    private double weight;
    private int reps;

    public StrengthExercise(String name, int duration, double weight, int reps) {
        super(name, duration);
        this.weight = weight;
        this.reps = reps;
    }

    @Override
    public double calcCalories() {
        return weight * reps * 0.1;
    }

    @Override
    public String showInfo() {
        return "[Strength] " + name + " - " + duration + "min - " + weight + "kg - " + reps + "reps";
    }
}

File: Python/classes/strength_exercise.py
from classes.exercise import Exercise

class StrengthExercise(Exercise):
    
    def __init__(self, name, duration, weight, reps):
        super().__init__(name, duration)
        self.weight = weight
        self.reps = reps

    def calc_calories(self):
        return self.weight * self.reps * 0.1

    def show_info(self):
        return "[Strength] " + self.name + " - " + str(self.duration) + "min - " + str(self.weight) + "kg - " + str(self.reps) + "reps"